lms/views/api/error.py

```diff
@@ -7,11 +7,19 @@
 )
 
 from lms.services import CanvasAPIError, CanvasAPIAccessTokenError
-
+from lms.validation import ValidationError
 
 _ = i18n.TranslationStringFactory(__package__)
 
 
+@exception_view_config(context=ValidationError, renderer="json")
```

Comment:
This didn't exist before and was needed to make usefully-worded errors for bad payloads.

Reply:
:+1: